app/views/katello/api/v2/host_collections/_host_collection.json.rabl

```diff
@@ -1,4 +1,5 @@
 attribute :pulp_id, :name, :organization_id, :max_hosts, :description, :total_hosts, :unlimited_hosts
+attributes :host_ids
```

Comment:
@jlsherrill, We can look at doing that; however, this was a regression raised by QE.